client/webserver/site/src/js/ws.ts

```diff
@@ -128,7 +136,12 @@ class MessageSocket {
         clearTimeout(timeout)
         if (retrys > 0) {
           retrys = 0
-          reloader()
+          // Once dexc is back online we have to reload book/candles (and maybe other
+          // stuff) otherwise we'll be missing a bunch of data for display in UI.
+          // Note, reloading page like this will result in ditching this WS connection
+          // and reestablishing new one.
+          reloadPage()
+          return
```

Comment:
Since we are reloading app here, it seems we can just return and drop those requests being served from the queue below ? Or maybe page reload does return there anyway, just explicitly stating that's what the intent is here with `return` probably doesn't hurt (those queued requests might not execute before WS connection is closed due to page refresh).